students/alexLaws/lesson03/slicing_Lab.py

```diff
@@ -0,0 +1,45 @@
+def exchange_first_last(seq):
```

Comment:
you want your py file names to be all lower case separated by the underscores -> `slicing_lab.py`